tools/compression_advisor.py

```
@@ -0,0 +1,1012 @@
+            "WHERE keyspace_name = %s AND table_name = %s",
+            (keyspace, table),
+        )
+        row = rows.one()
+        if row is None:
+            print(f"Error: table {keyspace}.{table} not found", file=sys.stderr)
+            sys.exit(1)
+        # A table may have an empty/absent compression map (uncompressed);
+        # normalize to {} so the caller falls back to default settings rather
+        # than crashing on dict(None).
+        return dict(row.compression or {})
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)